src/pyop/authz_state.py

```diff
@@ -78,9 +82,11 @@ def __init__(self, subject_identifier_factory, authorization_code_db=None, acces
         Mapping of user id's to subject identifiers.
         """
         self.subject_identifiers = subject_identifier_db if subject_identifier_db is not None else {}
+        self.stateless = stateless
 
-    def create_authorization_code(self, authorization_request, subject_identifier, scope=None):
-        # type: (AuthorizationRequest, str, Optional[List[str]]) -> str
+    def create_authorization_code(self, authorization_request, subject_identifier, scope=None, user_id=None,
+                                  user_info=None):
```

Comment:
What does `user_id` and `user_info` hold?

`user_id` and `subject_identifier` look the same.

Reply:
- user_id is a local user identifier which means that it is also the key used to look up the user_db.
- user_info is that user_db holding all the user_claims
- I debugged and indeed the value of user_id and subject_identifier was the same.

I have made the respective changes accordingly.